src/components/Tutorials/NewTutorial/index.jsx

```diff
@@ -252,7 +252,7 @@ const NewTutorial = ({ viewModal, onSidebarClick, viewCallback, active }) => {
                 loading={loading}
                 onClick={e => onSubmit(e)}
                 data-testid="newTutorialSubmit"
-                style={{ backgroundColor: "#03AAFA", borderRadius: "30px" }}
+                style={{ backgroundColor: "#03AAFA", borderRadius: "30px", color: "#FFFFFF" }}
```

Comment:
can you use `@mui/colors` instead of hardcoded colors? 
Reference : https://mui.com/material-ui/customization/color/#2014-material-design-color-palettes
Also optionally, you can update it to the mui sx prop (Reference: https://mui.com/system/getting-started/the-sx-prop/)

Reply:
sure